src/nvim/grid_defs.h

```diff
@@ -10,7 +10,7 @@
 #define MAX_MCO  6  // fixed value for 'maxcombine'
 
 // The characters and attributes drawn on grids.
-typedef char_u schar_T[(MAX_MCO + 1) * 4 + 1];
+typedef char schar_T[(MAX_MCO + 1) * 4 + 1];
```

Comment:
Changing `schar_` functions to also use `char *` can probably reduce a lot of casts.